packages/contracts/evm-contracts/contracts/token/InverseAppProjectedNft.sol

```diff
@@ -95,8 +95,8 @@ contract InverseAppProjectedNft is IInverseAppProjectedNft, ERC721, Ownable {
         return tokenId;
     }
 
-    function mint(address _to) external returns (uint256) {
-        return this.mint(_to, "");
+    function mint(address _to) public returns (uint256) {
```

Comment:
This no longer matches the interface

Reply:
Not true, functions in interfaces **must** be declared external, implementation can be declared public.